docs/source/rest-api/openapi.yml

```
@@ -0,0 +1,166 @@
+openapi: 3.0.0
+servers:
+  - description: Development IMAP SDC Server
+host: https://api.dev.imap-mission.com/
+info:
+  version: "0.1.0"
+  title: IMAP SDC API
+  description: Describes the API endpoints for interacting with the IMAP SDC
+paths:
+
+  '/upload/{filename}':
```

should we change to this since it has parameter information right below it? It could confuse `filename` with endpoint.
```suggestion
  '/upload':
```